worlds/lufia2ac/Locations.py

```diff
@@ -0,0 +1,22 @@
+from typing import Dict, List, NamedTuple, Optional
+
+from BaseClasses import Location
+
+start_id: int = 0xAC0000
+
+
+class L2ACLocation(Location):
+    game: str = "Lufia II Ancient Cave"
+
+
+class LocationData(NamedTuple):
+    region: str
+    name: str
+    code: Optional[int]
+
+
+l2ac_location_table: List[LocationData] = [
+    LocationData("AncientDungeon", f"Blue chest {i + 1}", i) for i in range(256)
```

Comment:
seems a lot more location IDs are specified than available to use?

Reply:
you also don't seem to use this list or class anywhere outside of this definition, so i would recommend just completely removing this and only doing the dictionary comprehension with enumeration.